ports/foonathan-lexy/portfile.cmake

```
@@ -4,7 +4,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO foonathan/lexy
     REF "v${VERSION}"
```

Remove line from the script. It is obsolet.

Add
~~~cmake
string(REGEX REPLACE "^([0-9]+)[.]([0-9][.])" "\\1.0\\2" LEXY_VERSION "${VERSION}")
~~~
at the top and change
```suggestion
    REF "v${LEXY_VERSION}"
```
to deal with the version scheme quirks.

(As implemented in port poppler.)